.travis.yml

```diff
@@ -4,7 +4,6 @@ sudo: false
 
 # Early warning system to catch if Rubygems breaks something
 before_install:
-  - gem update --system
```

Comment:
We'd like to keep this for the reason listed in the comment. Otherwise travis isn't going to be running the latest rubygems. Can you add this back and let's look at fixing whatever error this was causing?